contributing/index.md

```diff
@@ -13,17 +13,43 @@ Contributions are heavily appreciated. It's the community that makes the group t
 4. Each section has its own contribute template that needs to be followed. So if you would like to add a field under `Learning Resources`, `Career Paths`, then go ahead the copy the format from their `README.md` and you can start writing
 5. The MISC folder in `Learning Resouces` is for very niche topics, not exactly whole fields.
 
-## Sending a Pull Request
+## Sending A Pull Request
```

Comment:
I think rather than explaining the process ourselves we can copy content and images from here: https://github.com/firstcontributions/first-contributions

Reply:
You're right. The link you provided has a much more in depth explanation and step by step procedure. Would it be okay to say that we should add a new file separately for sending a pull request, and link that in contributing.md?

The new content's location would be, `contributing/first-pull-request.md`, which basically just copies the same content over from the repo's README.md which you provided.
Only real work would be moving the images to assets and figuring out their paths, which shouldn't be too difficult.

Reply:
Yeapp sounds like a good idea.

Reply:
Just committed and pushed a change that removes this section, and just links to the `first-pull-request.md` that was created under `contributing`. 

See PR #20